yarn.lock

```diff
@@ -3404,11 +3404,15 @@ eslint@^8.10.0, eslint@^8.11.0, eslint@^8.2.0:
     text-table "^0.2.0"
     v8-compile-cache "^2.0.3"
 
+<<<<<<< HEAD
```

Comment:
This file has Git conflicts that need to be resolved.

Reply:
@Basultobd I pulled the new changes of the base repo and it already had these conflicts. I'm not sure which is the correct change, I think that it'll be better that the person who commited the conflicts fix it.

Reply:
You're right, that actually comes from a PR that I merged, I'm going to fix that and let you know when I finished.

Reply:
The fix for this issue is already in master, you can rebase your PR :)